x/staking/types/params.go

```diff
@@ -33,16 +33,22 @@ const (
 
 // DefaultMinCommissionRate is set to 0%
 var DefaultMinCommissionRate = math.LegacyZeroDec()
+var DefaultMaxCommissionRate = math.LegacyNewDecWithPrec(25, 2)
```

Comment:
do we plan to override this in v4?

Reply:
nope 25% was proposed. made it default here